cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -68,6 +69,36 @@ function LogViewerController ($scope, LogViewerStore) {
     }
   };
 
+  myLogsApi.nextLogsJson({
+    'namespace' : this.namespaceId,
+    'appId' : this.appId,
+    'programType' : this.programType,
+    'programId' : this.programId,
+    'runId' : this.runId
+  }).$promise.then(
+    (res) => {
+      angular.forEach(res, (element, index) => {
+        if(res[index].log.logLevel === 'WARN'){
+          this.warningCount++;
+        } else if(res[index].log.logLevel === 'ERROR'){
+          this.errorCount++;
```

Comment:
we are getting these counts from Timeline query as well right? I believe that should be the source of truth, since that is the full count for the entire timeline, instead of only the existing logs in the table